File: backend/DataAccess/DaoGenerated/システム共有/tt_afkokuhoDto.cs
// *******************************************************************
// 業務名称　: 健康管理システム
// 機能概要　: DTO定義
//             【国民健康保険】国保資格情報テーブル
// 作成日　　: 2023.01.23
// 作成者　　: 屠
// 変更履歴　:
// *******************************************************************
namespace BCC.Affect.DataAccess
{
    public class tt_afkokuhoDto : DaEntityModelBase
    {
		public const string TABLE_NAME = "tt_afkokuho";
		public string atenano { get; set; }                       //宛名番号
		public string sikutyosonhokenjano { get; set; }           //市区町村保険者番号
		public string? hokenjanm { get; set; }                    //保険者名称
		public string kokuho_kigono { get; set; }                 //国保記号番号
		public string kokuho_edano { get; set; }                  //枝番
		public string kokuho_sikakukbn { get; set; }              //国保資格区分
		public string kokuho_sikakusyutokuymd { get; set; }       //国保資格取得年月日
		public string kokuho_sikakusyutokujiyu { get; set; }      //国保資格取得事由
		public string? kokuho_sikakusosituymd { get; set; }       //国保資格喪失年月日
		public string? kokuho_sikakusositujiyu { get; set; }      //国保資格喪失事由
		public string kokuho_tekiyoymdf { get; set; }             //国保適用開始年月日
		public string? kokuho_tekiyoymdt { get; set; }            //国保適用終了年月日
		public string syokbn { get; set; }                        //証区分
		public string yukokigenymd { get; set; }                  //有効期限
		public string? marugakumaruenkbn { get; set; }            //マル学マル遠区分
		public string reguserid { get; set; }                     //登録ユーザーID
		public DateTime regdttm { get; set; }                     //登録日時
		public string upduserid { get; set; }                     //更新ユーザーID
		public DateTime upddttm { get; set; }                     //更新日時
    }
}

File: backend/BusinessService/Db/DaoGenerated/システム共有/tt_afkokuhoDto.vb
' *******************************************************************
' 業務名称　: 互助防疫システム
' 機能概要　: DTO定義
'             【国民健康保険】国保資格情報テーブル
' 作成日　　: 2024.07.23
' 作成者　　: 
' 変更履歴　:
' *******************************************************************
Namespace JBD.GJS.Db
    Public Class tt_afkokuhoDto
        Inherits DaEntityModelBase
        Public Const TABLE_NAME As String = "tt_afkokuho"
        Public Property atenano As String                       '宛名番号
        Public Property sikutyosonhokenjano As String           '市区町村保険者番号
        Public Property hokenjanm As String                    '保険者名称
        Public Property kokuho_kigono As String                 '国保記号番号
        Public Property kokuho_edano As String                  '枝番
        Public Property kokuho_sikakukbn As String              '国保資格区分
        Public Property kokuho_sikakusyutokuymd As String       '国保資格取得年月日
        Public Property kokuho_sikakusyutokujiyu As String      '国保資格取得事由
        Public Property kokuho_sikakusosituymd As String       '国保資格喪失年月日
        Public Property kokuho_sikakusositujiyu As String      '国保資格喪失事由
        Public Property kokuho_tekiyoymdf As String             '国保適用開始年月日
        Public Property kokuho_tekiyoymdt As String            '国保適用終了年月日
        Public Property syokbn As String                        '証区分
        Public Property yukokigenymd As String                  '有効期限
        Public Property marugakumaruenkbn As String            'マル学マル遠区分
        Public Property reguserid As String                     '登録ユーザーID
        Public Property regdttm As Date                     '登録日時
        Public Property upduserid As String                     '更新ユーザーID
        Public Property upddttm As Date                     '更新日時
    End Class
End Namespace
